generative_ai/context_caching/update_context_cache.py

```diff
@@ -36,3 +36,10 @@ def update_context_cache(project_id: str, cache_id: str) -> str:
     # [END generativeaionvertexai_gemini_update_context_cache]
 
     return cached_content.expire_time
+
+
+if __name__ == "__main__":
+    import os
```

Comment:
Please move `import` and `PROJECT_ID` to the top portion of the file.

Reply:
Done